schedule/management/commands/create_downtime.py

```diff
@@ -51,9 +53,10 @@ def handle(self, *args, **options):
             site=options['site'],
             enclosure=options['enclosure'],
             telescope=options['telescope'],
+            instrument_type=options['instrument_type'],
             reason=options['reason'],
             start=start,
             end=end
         )
-        logger.info(f"Created downtime on {options['site']}.{options['enclosure']}.{options['telescope']} at {start}")
+        logger.info(f"Created downtime on {options['site']}.{options['enclosure']}.{options['telescope']}.{options['instrument_type']} at {start}")
```

Comment:
If the instrument type is empty, then this resource will have a trailing dot. I would say handle that case, maybe leaving out the trailing dot when there is no instrument type.

Reply:
I knew you would catch this! I figured it was fine since its just a log line for a management command, but I was just being lazy